low-level/files/opt/seagate/sspl/conf/sspl.config.conf.tmpl

```diff
@@ -0,0 +1,45 @@
+cortx:
+  release:
+    product: LR2
+    setup: cortx
+server_node:
+  30512e5ae6df9f1ea02327bab45e499d:
```

Comment:
server_node: 
    TMPL_MACHINE_ID: 

This is a template. Do not fill in the values yourself. Imagine you fill up any application form.

Reply:
Sure. I corrected them.
Few fields are constants like below. Should I keep the value for them or change to TEMP_XXXX?

sspl.init.tmpl.1-node
```
cluster:
  cid01:
    name: cortx_cluster
    site:
      storage_set_count: '1'
    site_count: '1'
    storage_set:
    - name: StorageSet-1
      server_nodes:
      - srvnode-1
      storage_enclosures:
      - encserial-1
cortx:
  release:
    product: LR2
    setup: cortx
```